src/store.js

```diff
@@ -0,0 +1,79 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+import axios from '@/axiosInstance';
+
+Vue.use(Vuex);
+
+export default new Vuex.Store({
+  state: {
+    status: '',
+    token: localStorage.getItem('token') || '',
```

Comment:
maakt de || het hier geen condition die true of false teruggeeft? Het lijkt erop dat zolang er een token bestaat die "true" returned

Reply:
Het geeft inderdaad aan als er een token bestaat  dat ie true moet terug geven. Met ` || ''` geeft die false terug. Is inmiddel veranderd naar false